### PR TITLE
Update CHANGELOG for 9.2.0

### PR DESCRIPTION
**Pull request by onurctirtir**
Update CHANGELOG for 9.2.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,47 @@
...
+
+* Disallows marking ref. table shards unhealthy in the presence of savepoints
+
+* Disallows placing new shards with shards in TO_DELETE state
+
+* Enables local execution of queries that do not need any data access
+
+* Fixes Makefile trying to cleanup PG directory during install
+
+* Fixes a bug causing errors when planning a query with multiple subquerie
```
Comment (JelteF) [inline CHANGELOG.md]:
```suggestion
* Fixes a bug causing errors when planning a query with multiple subqueries
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,47 @@
...
+
+* Fixes a possible deadlock that could happen during shard moves
+
+* Fixes a problem when adding new node due to tables referenced in func body
```
Comment (JelteF) [inline CHANGELOG.md]:
```suggestion
* Fixes a problem when adding a new node due to tables referenced in func body
```

**Comment by onurctirtir** [comment]
> I'm missing an entry for both of the CTE changes that @marcocitus mentioned yesterday. Can you check with his presentation that all of the ones in there that relate to community are also here?

For two CTE improvements:
For first CTE improvement:
 - Actually, `* Enables local execution of queries that do not need any data access` includes CTE local execution as well, so leaving this entry as is ?
For second CTE improvement:
 - Adding another now for this

File: configure [hunk 3]
```diff
@@ -992,6 +994,15 @@ do
   | -silent | --silent | --silen | --sile | --sil)
     silent=yes ;;
 
+  -runstatedir | --runstatedir | --runstatedi | --runstated \
```
Comment (onderkalaci) [inline configure]: why do we have these changes in this PR? Are they related to readme? If not, maybe opening a separate PR for those could be nicer.

Comment (JelteF) [reply configure]: These keep happening because we people have different versions of autoreconf.

Comment (onurctirtir) [reply configure]: I will revert this change

Comment (onurctirtir) [reply configure]: reverted

**Comment by metdos** [comment]
Can we pull "Adds support for `INSERT...SELECT` queries with re-partitioning" to the top?

**Comment by onurctirtir** [comment]
> Can we pull "Adds support for INSERT...SELECT queries with re-partitioning" to the top?

done

**Comment by onurctirtir** [comment]
also updated the date

**Comment by metdos** [comment]
Looks good to me

**Pull request merged**